modules/tinymce/src/plugins/quickbars/test/ts/module/test/Utils.ts

```diff
@@ -0,0 +1,15 @@
+import { UiFinder, Waiter } from '@ephox/agar';
+import { SugarBody } from '@ephox/sugar';
+
+const pAssertToolbarNotVisible = async (): Promise<void> => {
+  await Waiter.pWait(50);
+  UiFinder.notExists(SugarBody.body(), '.tox-pop__dialog .tox-toolbar');
+};
+
+const pAssertToolbarVisible = (): Promise<void> =>
+  Waiter.pTryUntil('toolbar should exist', () => UiFinder.exists(SugarBody.body(), '.tox-toolbar'));
```

Comment:
Sorry Ben you beat me to it as I accidentally submitted the comment... I meant to add this as well otherwise you could pickup the regular toolbar:
```suggestion
  Waiter.pTryUntil('toolbar should exist', () => UiFinder.exists(SugarBody.body(), '.tox-pop__dialog .tox-toolbar'));
```

Reply:
This was like this previously though, so not caused by any change you made.

Reply:
No worries.